ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/legaladvisor/CCDOrderDirection.java

```diff
@@ -1,15 +1,47 @@
 package uk.gov.hmcts.cmc.ccd.domain.legaladvisor;
 
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateSerializer;
 import lombok.Builder;
 import lombok.Value;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
 
 import java.time.LocalDate;
+import java.util.List;
 
 @Value
 @Builder
+@JsonTypeName(value = "OtherDirection")
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class CCDOrderDirection {
+
     private CCDOrderDirectionType extraOrderDirection;
-    private String otherDirection;
+
+    private String directionComment;
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy")
+    @JsonSerialize(using = LocalDateSerializer.class)
     private LocalDate sendBy;
+
     private CCDDirectionPartyType forParty;
+
+    private List<CCDCollectionElement<String>> extraDocUploadList;
+
```

Comment:
Hi @simad , spoke to @SanchitaMoj  regarding the naming. She said these are free form text written by judges as checklist to submit expert reports. This field name should reflect actual usage. Can we pls change it? 
Suggestions - reportCheckList / expertReportCheckList / judgesCheckList / reportReviewsByJudges / judgesExpertReportReview  
Sanchita's suggestion 
thanks